schema/gosling.schema.json

```diff
@@ -415,9 +415,14 @@
         },
         "scaleOffset": {
           "description": "Whether to use offset of the domain proportionally. This is bound to brushes on the color legend. __Default__: `[0, 1]`",
-          "items": {
-            "type": "number"
-          },
+          "items": [
```

Comment:
Maybe double-check to run `yarn`? I recently upgraded `ts-json-schema-generator`, and I wonder if the local version on your machine is the old one that makes the resulting schema different.